### PR TITLE
Importer number alignment

### PR DESCRIPTION
**Pull request by nissimsan**
This aligns 7501 with how IOR is defined on 3461 and CTPAT.

### CODE REVIEW

File: docs/openapi/components/schemas/common/CBPEntrySummary.yml [hunk 1]
```diff
@@ -199,10 +199,10 @@ properties:
   importerNumber:
     title: Importer Identifier
     description: Record the IRS EIN, SSN, or CBP assigned number of the importer of record. Proper format is listed under the instructions for Consignee Number.
-    type: string
+    $ref: ./CBPImporterOfRecord.yml
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/CBPEntrySummary.yml:202]: Why would we do this? 
This is a CBP entry summary when would you not have the importerNumber be the CBPImporter of Record?

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/CBPEntrySummary.yml [hunk 2]
```diff
@@ -199,10 +199,10 @@ properties:
...
     $linkedData:
       term: importerNumber
-      '@id': https://schema.org/identifier
+      '@id': https://w3id.org/traceability#importerOfRecord
```
Comment (OR13) [inline docs/openapi/components/schemas/common/CBPEntrySummary.yml:205]: sad that we define this

**Comment by OR13** [comment]
I'm fine merging this outside of call time.

**Comment by nissimsan** [comment]
@nissimsan , merge when sorted out with @mkhraisha

File: docs/openapi/components/schemas/common/CBPEntrySummary.yml (hunk 2 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/CBPEntrySummary.yml:205]: Good point. 

How about `Declarant's Customs identity number`: https://vocabulary.uncefact.org/ReferenceCodeList#ABP

File: docs/openapi/components/schemas/common/CBPEntrySummary.yml (hunk 1 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/CBPEntrySummary.yml:202]: Indeed you have one of the applicable importer identifiers: IRS EIN, SSN, or CBP assigned. The class is to allow indication which one it is. We use it elsewhere, this is to make 7501 consistent.

**Pull request merged**